tools/install.sh

```diff
@@ -13,37 +26,42 @@ fi
 
 echo "--  Checking for application directory"
 applicationDir="${HOME}/.terra"
-mkdir -p $applicationDir
+mkdir -p "$applicationDir"
```

Comment:
in general it would nicer to make all these formatting changes separately, just to make it easier to review (and revert if needed) the logic change.

Reply:
yeah, I can either make IntelliJ happy or my co-workers happy...